src/_langs/en/updates/2015-07-22-access-ble-devices-with-web-bluetooth.markdown

```diff
@@ -0,0 +1,279 @@
+
+{% highlight javascript %}
+navigator.bluetooth.requestDevice({ filters: [{ services: ['heart_rate'] }] })
+.then(device => device.connectGATT())
+.then(server => server.getPrimaryService('heart_rate'))
+.then(service => service.getCharacteristic('heart_rate_control_point'))
+.then(characteristic => {
+  var resetEnergyExpended = new Uint8Array([1]);
+  return characteristic.writeValue(resetEnergyExpended);
+})
+.then(function(value) {
+  console.log('Energy expended has been reset.');
+})
+.catch(error => { console.log(error); });
+{% endhighlight %}
+
+## Samples
+
+Samples below have been tested on Chrome OS M45 with the Web Bluetooth flag
+enabled. To enjoy these samples to their fullest, I recommend you install the [BLE Peripheral Simulator Android
+App](https://play.google.com/store/apps/details?id=io.github.webbluetoothcg.bletestperipheral)
+which simulates a BLE Peripheral with a Battery Service or a Heart Rate
+Service.
+
+- Battery Level Sample - [https://googlechrome.github.io/samples/web-bluetooth](https://googlechrome.github.io/samples/web-bluetooth)
+- Reset Energy Expended Sample - [https://scheib.github.io/bluetooth-demos/heart-rate-write.html](https://scheib.github.io/bluetooth-demos/heart-rate-write.html)
+- Device Info Sample - [https://scheib.github.io/bluetooth-demos/device-info.html](https://scheib.github.io/bluetooth-demos/device-info.html)
+
+## Dev Tip
+
+A Bluetooth Console is available in Chrome OS developer shell. Press [ Ctrl ] [
+Alt ] [ T ] to open a browser tab terminal and use the `bt_console` command to
+start poking around your bluetooth settings. The `help` command will give you a
+list of all available commands.
+
+<img src="/web/updates/images/2015-07-22-access-ble-devices-with-web-bluetooth/bluetooth-developer-console.png" alt="Bluetooth Developer Console screenshot"/>
+
+## What's next
+
+As the Web Bluetooth API implementation is not complete yet, here's a sneak
+peek of what to expect in the coming months:
+
+- Bluetooth GATT Characteristics will support `startNotifications` and
+  `stopNotifications` functions to [subscribe to
+  notifications](https://webbluetoothcg.github.io/web-bluetooth/#notification-events)
+  from Bluetooth Devices.
+- Connecting and disconnecting a Bluetooth Device will be possible with
+  new `BluetoothDevice.connect()` and `BluetoothDevice.disconnect()` functions.
+- A new `serviceadded` event will track newly discovered Bluetooth GATT Services
+  while `serviceremoved` event will track removed ones. A new `servicechanged`
+  event will fire when any characteristic and/or descriptor gets added or
+  removed from the Bluetooth GATT Service.
+- A Promise to [detect if Bluetooth is available on the
+  platform](https://github.com/WebBluetoothCG/web-bluetooth/issues/127) will be
+  added to improve user experience.
+
+At the time of writing, Chrome OS M45 is [the most advanced
```

Comment:
Perhaps link here to implementation status instead.https://github.com/WebBluetoothCG/web-bluetooth/blob/gh-pages/implementation-status.md